### PR TITLE
[UnitTests][CodeRefactor] Simple unit test for Rename command

### PR DESCRIPTION
**Pull request by SlavaRa**
@Neverbirth please review this PR

### CODE REVIEW

**Comment by SlavaRa** [comment]
@Neverbirth ?

File: Tests/External/Plugins/CodeRefactor.Tests/Commands/RefactorCommandTests.cs [hunk 1]
```diff
@@ -350,5 +355,83 @@ internal static string Common(ScintillaControl sci, string sourceText, string fi
...
+                        .CreateRenameCommandAndExecute(RefactoringHelper.GetDefaultRefactorTarget(), false, newName)
+                        .OnRefactorComplete += (sender, args) => waitHandle.Set();
+                const int waitTime = 2;
+                var end = DateTime.Now.AddSeconds(waitTime);
+                var result = false;
+                while ((!result) && (DateTime.Now < end))
+                {
+                    Application.DoEvents();
+                    result = waitHandle.WaitOne(0);
+                }
```
Comment (Neverbirth) [inline Tests/External/Plugins/CodeRefactor.Tests/Commands/RefactorCommandTests.cs]: A bit hacky and would be better to slightly improve some parts of logic, but hacky tests are way better than no tests at all.

**Comment by Neverbirth** [comment]
This broke the normal build.

File: Tests/External/Plugins/CodeRefactor.Tests/Commands/RefactorCommandTests.cs (hunk 1 above)
Comment (Neverbirth) [reply Tests/External/Plugins/CodeRefactor.Tests/Commands/RefactorCommandTests.cs]: `Application.DoEvents()` doesn't work for console type applications.

Comment (Neverbirth) [reply Tests/External/Plugins/CodeRefactor.Tests/Commands/RefactorCommandTests.cs]: I've marked the test to be ignored in the meantime to be able to get builds.

**Comment by SlavaRa** [comment]
Wow, I will check it tomorrow

**Pull request merged**